nvflare/fuel/hci/server/binary_transfer.py

```diff
@@ -41,10 +41,11 @@ def download_folder(self, conn: Connection, tx_id: str, folder_name: str):
         engine = conn.get_prop(ConnProps.ENGINE)
         cell = engine.get_cell()
         source_fqcn = cell.get_fqcn()
-        download_tid = FileDownloader.new_transaction(
+        downloader = ObjectDownloader(
+            num_receivers=1,
             cell=engine.get_cell(),
             timeout=5,
```

Comment:
a user points out this should be configurable in https://github.com/NVIDIA/NVFlare/issues/3853